FF1Lib/Hacks.cs

```diff
@@ -537,6 +537,11 @@ public void EnableFreeLute()
 			Data[0x3020 + (int)Item.Lute] = 0x01;
 		}
 
+		public void EnableFreeTail()
+		{
+			Data[0x3020 + (int)Item.Tail] = 0x01;
+		}
+

```

Comment:
This isn't quite good enough - and finishing it off will be easy. Look for like, "FreeAirship" elsewhere in the code and flesh it out so that if the TAIL is free it's replaced with a cabin, and no longer incentivized, and that state is correctly represented in the UI.

I think you'll just need to edit IncentivesData.cs and ItemPlacement.cs

The rest of this patch is pretty great through.

Reply:
I put in the changes outlined below for "FreePromotion".  It works properly on the web page when counting key items/incentives, and as far as I know it should mean the tail is properly incentivized.

Also added some tweaks for balance / quality of life.